src/components/settings/Settings.h

```diff
@@ -11,7 +12,12 @@ namespace Pinetime {
     public:
       enum class ClockType { H24, H12 };
       enum class Vibration { ON, OFF };
-      enum class WakeUpMode { None, SingleTap, DoubleTap, RaiseWrist };
+      enum class WakeUpMode : uint8_t {
+        None = 0,
```

Comment:
I don't think `None` needs to be its own enumerated value here. `None` should effectively just be equivalent to all three other options disabled. This would allow getting rid of the extra logic to clear the None checkbox when other are selected or set it when no other modes are enabled.

Reply:
Shouldn't be a problem anymore.